apps/matrix-rain-showcase/src/components/demo/demo.css

```
@@ -0,0 +1,369 @@
+.rail-console-empty {
+  margin: 0;
+  color: var(--text-dim);
+  font-style: italic;
+}
+
+.rail-console-line {
+  margin: 0 0 4px;
+  display: grid;
+  grid-template-columns: 64px 1fr;
+  gap: 6px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)